DEPS

```diff
@@ -942,11 +945,22 @@ deps = {
     'dep_type': 'cipd',
   },
 
+  'src/buildtools/windows-x64/reclient': {
+    'packages': [
+      {
+        'package': 'infra/rbe/client/${{platform}}',
+        'version': Var('reclient_version'),
+      }
+    ],
+    'condition': 'use_rbe and download_windows_deps',
```

Comment:
Do you need `host_cpu == "x64"`?

Reply:
We don't currently do that anywhere else for Windows dependencies, so I'll follow suit in this PR.